handlers/UpdateHandler__.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace YesWiki\Shop;
+
+use Configuration;
+use YesWiki\Bazar\Service\FormManager;
+use YesWiki\Core\YesWikiHandler;
+use YesWiki\Security\Controller\SecurityController;
+
+class UpdateHandler__ extends YesWikiHandler
+{
+    private const PATHS = [
+        'forms' => [
+            'Produit' => 'tools/shop/setup/forms/Form - Produit.txt',
+        ],
+    ];
+
+    public function run()
+    {
+        if ($this->getService(SecurityController::class)->isWikiHibernated()) {
+            throw new \Exception(_t('WIKI_IN_HIBERNATION'));
+        };
+        if (!$this->wiki->UserIsAdmin()) {
+            return null;
+        }
+
+        $formIdsParam = $this->params->get('shop')['forms']['products'] ?? '';
+        $productsFormIds = array_filter(array_map('trim', explode(',', $formIdsParam)));
+
+        if (empty($productsFormIds)) {
+            $formId = $this->createDefaultForm();
+            $this->updateWakkaConfig($formId);
+            return;
+        }
+
+        foreach ($productsFormIds as $productsFormId) {
+            $this->getFormOrCreate($productsFormId);
+        }
+
+        return null;
+    }
+
+    /**
+     * @param string $formId
+     */
+    private function getFormOrCreate(string $formId): void
```

Comment:
```suggestion
     * @return string $formId if created
     */
    private function getFormOrCreate(string $formId): string
```
l'idée est de retourner un message d'information